### ISSUE TITLE
USER ISSUE: Banned from my own server "?!"

### ISSUE THREAD

**Issue by Phlo123** [StrangeLoopGames/EcoIssues#9694]
**Version:** 0.7.7.2 beta

**Steps to Reproduce:**
Open Game scene in unity, hit play, log in and try to connect to your 1000 euro server you bought with your own money. Works on other server that you don't own, but its better if you've spent the money. If I log in with a normal client it works fine.

**Expected behavior:**
It should let me log in. lol

**Actual behavior:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

You have been banned from this server
```n
**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

You have been banned from this server
```

**Comment by Phlo123** [comment]
![image](https://user-images.githubusercontent.com/17730996/46923238-cefb3a00-d01d-11e8-86c0-74f7bd4391b2.png)

**Comment by elfl0rd** [comment]
um...
Unity? 0.7.7.2? what branch you using?
You asked the host from whom you rent the server?
If you have file acces, you checked Users.eco?

**Comment by Phlo123** [comment]
Sorry i didn't respond sooner.
I opened Game Scene in unity and hit play, it boots up the game.
I'm using Release branch.
I don't host the server anywhere, I own the machine and the network it uses. 
I checked Users file and my slg id was under blacklist, and also under Admin(but I gave myself Admin).

**Comment by elfl0rd** [comment]
Yes, you can shot to your leg by adding self to blacklist.

**Comment by pamduncan** [comment]
I have had this happen one time when connecting to a remote computers when the game was launch  through Unity.  It also added me to the blacklist.  I All I could think is it was some part of the code that adds you automatically as an admin when playing the single player world and got the wrong list?  I removed my ID from the blacklist and it worked fine and has not happened again, and I could not make it happen again so, since I could not repro it I did not post it, but I know I did add my ID to the admin but did not blacklist myself so I have no idea how my ID got in the blacklist
